source/Subsystem/Renderer/RenderContext.cpp

```diff
@@ -82,21 +89,27 @@ void RenderContext::CreateBuffers(std::vector<Model>& aModels)
 		glEnableVertexAttribArray(1);
 		glVertexAttribPointer(1, 2, GL_FLOAT, GL_FALSE, sizeof(Vertex), (GLvoid*)offsetof(Vertex, myTextureCoordinates));
 
+		glEnableVertexAttribArray(2);
+		glVertexAttribPointer(2, 3, GL_FLOAT, GL_FALSE, sizeof(Vertex), (GLvoid*)offsetof(Vertex, myNormal));
+
+		glEnableVertexAttribArray(3);
+		glVertexAttribPointer(3, 3, GL_FLOAT, GL_FALSE, sizeof(Vertex), (GLvoid*)offsetof(Vertex, myTangent));
+
```

Comment:
C-Style casts?!!?!?!?!?!

Reply:
Tangent currently is unused in the shaders but will be necesarry for normal mapping!

Reply:
True, we can actually calculate and pass the tangents and bitangents without much work.